go.mod

```diff
@@ -54,6 +54,90 @@ require (
 	k8s.io/client-go v0.20.6
 )
 
+require (
```

Comment:
Don't like this double require here. Unless anyone has a reason why this is good it should be removed.

There is also https://github.com/golang/go/issues/46144 issue. I'm not sure if it affects projects importing buildx. If it does then we should wait with updating gomod. It doesn't seem to buy us anything anyway as the removed go.mod files do not affect binary size.

Reply:
> Don't like this double require here. Unless anyone has a reason why this is good it should be removed.

With this second require we can see all transitively-imported packages so I tend to like it but I'm not against to merge.

> There is also golang/go#46144 issue.

:+1:

Reply:
Except there are a lot of indirects in the first block as well. Is there any guidance from go project for how this should be dealt with?

Reply:
No guidance afaik so I guess we can merge them.

Reply:
https://github.com/moby/buildkit/pull/2331